README.md

```diff
@@ -28,7 +28,20 @@ To see Blazor in action, check out [Steve Sanderson's prototype demo at NDC Oslo
 
 ## Getting Started
 
-We are still *very early* in this project. There isn't yet anything you can download nor any project template you can use. Most of the planned features aren't implemented yet. Even the parts that are already implemented aren't yet optimized for minimal payload size. If you're keen, you can clone the repo, build it, and run the samples and tests.
+1. Install the [.NET Core 2.1 Preview 1 SDK](https://www.microsoft.com/net/download/dotnet-core/sdk-2.1.300-preview1).
```

Comment:
I've tried the commandline shown below however the --nuget-source switch is not recognised ?

Invalid input switch:
  --nuget-source
  https://dotnet.myget.org/F/blazor-dev/api/v3/index.json
Run dotnet new --help for usage information.
See https://aka.ms/dotnet-install-templates to learn how to install additional template packs.

Reply:
This happened to me and was resolved when I installed .NET Core  2.1.300-preview1-008174